core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp

```diff
@@ -318,13 +314,7 @@
           {
             "refName": "yarn-GATEWAY-BASE",
             "roleType": "GATEWAY",
-            "base": true,
-            "configs": [
-              {
-                "name": "mapreduce_client_env_safety_valve",
```

Comment:
why we deleted this?

Reply:
We deleted this since it needs to be only set for AWS.